ldms/src/ldmsd/ldmsd.c

```diff
@@ -2391,6 +2383,15 @@ int main(int argc, char *argv[])
 		ldmsd_inband_cfg_mask_add(0777);
 	}
 
+	/* Check for at least a listening port */
+	struct ldmsd_listen *_listen;
+	_listen = (ldmsd_listen_t) ldmsd_cfgobj_first(LDMSD_CFGOBJ_LISTEN);
+	if (!_listen && !has_config_file) {
+		ldmsd_log(LDMSD_LCRITICAL,
+			"No config files nor listening ports are given ... exiting\n");
```

Comment:
@narategithub please revise log message to
"A config file (-c)  or listening port (-x) is required. Specify at least one of these. ... exiting\n");

Reply:
@tom95858 I previously proposed "A config file (-c) or listening port (-x) is required. Specify at least one of these. ... exiting\n"); to replace "No config files nor listening ports are given ... exiting\n");
How would you like it further improved?